workflow/models.py

```diff
@@ -125,6 +147,11 @@ class Organization(models.Model):
     chargebee_subscription_id = models.CharField(blank=True, null=True, max_length=50)
     chargebee_used_seats = models.IntegerField(blank=True, null=True, default=0)
     oauth_domains = fields.ArrayField(models.CharField("OAuth Domains", max_length=255, null=True, blank=True), null=True, blank=True)
+    date_format = models.CharField("Date Format", max_length=50, blank=True, default="DD.MM.YYYY")
+    default_currency = models.ForeignKey(Currency, blank=True, null=True)
+    currency_format = models.CharField("Currency Format", max_length=50, blank=True, default="Commas")
+    default_language = models.CharField("Default Language", max_length=100, blank=True, default="English")
```

Comment:
I think it'd be better to use [choices](https://docs.djangoproject.com/en/1.11/ref/models/fields/#choices) here. I know that it's gonna limit the number of languages, but we can always add more choices and it won't affect the database at all and we'll have a standard name for the languages instead of having things like `English`, `ENGLISH`, `english` and so on.
What do you think @Menda and @glind ?